setup.py

```diff
@@ -111,7 +111,7 @@
             'gcode_tempcal=gcodeutils.gcode_tempcal:main',
             'gcode_mod=gcodeutils.gcode_mod:main',
             'gcode_stretch=gcodeutils.gcode_stretch:main',
-            'gcode_arc_optimize=gcodeutils.gcode_optimize_arcs:main',
+            'gcode_optimize_arc=gcodeutils.gcode_optimize_arcs:main',
```

Comment:
I tend to keep the 's' on the CLI as it describes the function better.

'There are two hard things in computer science: cache invalidation, **naming things**, and off-by-one errors.'